operations/utility_storage/Troubleshoot_Pools_Have_Many_More_Objects_Per_Pg_Than_Average.md

```diff
@@ -0,0 +1,46 @@
+# Fixing incorrect number of PG Issues
+
+## Scenario 1
+
+**Symptom:** Ceph is reporting a `HEALTH_WARN` and the warning message is `1 pools have many more objects per pg than average`.
+
+**Cause:**
+
+>When we introduced an additional pool to the store the CSM artifacts we caused the amount of PGs created per OSD to not allow for the amount of data stored in that device.  This will typically be seen on 3 node Ceph clusters since our number of PGs is set by the number of OSDs.
```

Comment:
```suggestion
When an additional pool to the store the CSM artifacts was introduced, it caused the amount of PGs created per OSD to not allow for the amount of data stored in that device.
This will typically be seen on three node Ceph clusters since the number of PGs is set by the number of OSDs.
```